hack/make-docker-images.sh

```diff
@@ -38,11 +38,17 @@ while IFS= read -d $'\0' -r dir; do
         cd "${builddir}"
         log "Building: ${image}"
         docker build -t "${image}" .
-        docker build -t "${image}-native-grpc-probes" . --target without-grpc-health-probe-bin
 
         log "Pushing: ${image}"
         docker push "${image}"
-        docker push "${image}-native-grpc-probes"
+
+        if [ $svcname != "frontend" ] && [ $svcname != "loadgenerator" ]
+        then
+            log "Building: ${image}-native-grpc-probes"
+            docker build -t "${image}-native-grpc-probes" . --target without-grpc-health-probe-bin
+            log "Pushing: ${image}-native-grpc-probes"
+            docker push "${image}-native-grpc-probes"
+        fi
```

Comment:
Rationale for change: We need to make sure that we only build and push `-native-grpc-probes` for the gRPC microservices (i.e., every Online Boutique services except the `loadgenerator` and `frontend`).

Reply:
Yeah, I thought about that, but I will need to revisit the associated doc in this repo about that, otherwise the deployment of these 2 containers won't work. No worries, will do that later, it's an experimental feature anyway :)

Reply:
Following up on this, this https://github.com/GoogleCloudPlatform/microservices-demo/pull/1202 will fix that, JFYI.